Moroz_Homework_1.cpp

```diff
@@ -0,0 +1,59 @@
+#include <cmath>
+#include <iostream>
+#include <limits>
+
+constexpr uint16_t dividendMultiply(4);
```

Comment:
старайтесь использвать универсальную инициализацию
```suggestion
constexpr uint16_t dividendMultiply{4};
```